app/assets/stylesheets/govuk-component/_govspeak-html-publication.scss

```diff
@@ -117,7 +55,7 @@
       // Reduce default margin by 2x gutter for first headings
       // This accounts for the 2x gutter margin on the HTML pub header
       // which doesn't collapse because of containers
-      @include media(tablet) {
+      @include media(mobile) {
```

Comment:
Tweaked this so that media queries are mobile first. Start with a margin then reset for tablet and upwards.